package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "boxen",
-	"version": "5.0.1",
+	"version": "5.1.0",
```

Comment:
You should not bump version in a pull request. It's up to the maintainers to do so when it's time.

Reply:
> You should not bump version in a pull request. It's up to the maintainers to do so when it's time.

Ok didn't know that, figured a feature update should increment the second digit.